docs/components/BarChartDocs.js

```diff
@@ -0,0 +1,237 @@
+const React = require('react');
+
+const Markdown = require('components/Markdown');
+
+const { BarChart } = require('mx-react-components');
+
+const chartData = [
+  {
+    color: '#E3E6E7',
+    label: 'Jan',
+    value: 595.45
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Feb',
+    value: 690.65
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Mar',
+    value: 897.40
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Apr',
+    value: 1209.95
+  },
+  {
+    color: '#E3E6E7',
+    label: 'May',
+    value: 1191.00
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Jun',
+    value: 1391.27
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Jul',
+    value: 991.23
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Aug',
+    value: 1101.45
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Sep',
+    value: 1301.45
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Oct',
+    value: 1405.45
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Nov',
+    value: 1600.45
+  },
+  {
+    color: '#E3E6E7',
+    label: 'Dec',
+    value: 2100.45
+  }
+
+];
+
+const BarChartDocs = React.createClass({
+
+  getInitialState () {
+    return {
+      buttonIsActive: false,
+      spinnerIsActive: false
+    };
+  },
+
+  _handleButtonClick () {
+    this.setState({
+      buttonIsActive: !this.state.buttonIsActive
+    });
+  },
+
+  _handleSpinnerClick () {
+    this.setState({
+      spinnerIsActive: !this.state.spinnerIsActive
+    });
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <div>
+        <h1>
+          BarChart
+          <label>CHANGE THIS A D3 donut chart that supports a center label, multiple slices, and reference dots.</label>
```

Comment:
@jtanner updated the description but take a look and see if you would change it - i'm not  a huge fan of what I wrote